src/style.css

```diff
@@ -525,7 +524,6 @@ input::placeholder {
   ::-ms-input-placeholder {
     color: var(--text-color);
   }
-  caret-color: var(--text-color);
```

Comment:
seems you are adding your former code again, you deleting this deletes it from the code base.

close the PR, delete the fork, fork again and apply the properchanges

Reply:
will do it in some time